sellercenter-product-port/src/main/java/com/gfg/sellercenter/product/port/Product.java

```diff
@@ -0,0 +1,88 @@
+package com.gfg.sellercenter.product.port;
+
+import lombok.Data;
+import lombok.AllArgsConstructor;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.ZonedDateTime;
+
+@Data
+@AllArgsConstructor
+public class Product {
+    @NotNull
+    private final Integer id;
+
+    private final Integer sourceId;
+
+    private final String sku;
```

Comment:
Do we need also sellerSku?

Reply:
I thought that for the first version we can have it, as it's quite often used field, and it might save some effort from creating a new version when it will be needed.

Reply:
True, agree